src/components/v5/common/ActionSidebar/partials/UserSelect/hooks.ts

```diff
@@ -1,28 +1,52 @@
+import { useMemo } from 'react';
 import { useColonyContext } from '~hooks';
 import { useGetColonyMembers } from '~v5/shared/MembersSelect/hooks';
 import { UserSelectHookProps } from './types';
+import { useGetVerifiedMembersQuery } from '~gql';
+import { SearchSelectOption } from '~v5/shared/SearchSelect/types';
 
-export const useUserSelect = (): UserSelectHookProps => {
+export const useUserSelect = (inputValue: string): UserSelectHookProps => {
   const { colony } = useColonyContext();
-  const { members, loading } = useGetColonyMembers(colony?.colonyAddress);
+  const { colonyAddress = '', metadata } = colony ?? {};
+  const { members, loading } = useGetColonyMembers(colonyAddress);
+  const { data, loading: verifiedMembersLoading } = useGetVerifiedMembersQuery({
+    variables: { colonyAddress },
+    skip: !colonyAddress,
+  });
```

Comment:
Shouldn't this query be skipped when `isWhitelistActivated === false`  ?

Reply:
fixed